Add TiDB Dashboard documents (#2668)
pingcap/docs: dashboard/dashboard-statement-details.md

---

> title: Execution Details of TiDB Dashboard

Reviewer: ```suggestion
title: Statement Execution Details of TiDB Dashboard
```

summary: View the execution details of a single SQL statement in TiDB Dashboard.
category: how-to
---

> # Execution Details of TiDB Dashboard

Reviewer: ```suggestion
# Statement Execution Details of TiDB Dashboard
```

- Execution detail of plans, which displays the detailed information of the selected execution plans. See [Execution plan in details](#execution-plan-in-details) (area 3 in the image below).

![Details](/media/dashboard/dashboard-statement-detail.png)

## Execution details of plans

The execution detail of plans includes the following information:

- SQL sample: The text of a certain SQL statement that is actually executed corresponding to the plan. Any SQL statement that has been executed within the time range might be used as a SQL sample.
- Execution plan: For details of the execution plan, see [Understand the Query Execution Plan](/query-execution-plan.md). If multiple execution plans are selected, only (any) one of them is displayed.
- For basic information, execution time, Coprocessor read, transaction, and slow query of the SQL statement, you can click the corresponding tab titles to switch among different information.

![Execution details of plans](/media/dashboard/dashboard-statement-plans-detail.png)

> ### Basic information

Reviewer: ```suggestion
### Basic Tab
```

> The basic information of a SQL execution includes the table name, index name, execution times, and total duration. The **Description** column provides detailed description of each field.

Reviewer 2: To keep it consistent with the image.
```suggestion
The basic information of a SQL execution includes the table names, index name, execution count, and total latency. The **Description** column provides detailed description of each field.
```

![Basic information](/media/dashboard/dashboard-statement-plans-basic.png)

> ### Execution time

Reviewer: ```suggestion
### Time Tab
```

> Click the **Time** tab, and you can see how long each stage of the execution plan is lasted.

Reviewer 2: ```suggestion
Click the **Time** tab, and you can see how long each stage of the execution plan lasts.
```

> **Note:**
>
> Because some operations might be performed in parallel within a single SQL statement, the cumulative duration of each stage might exceed the actual execution time of the SQL statement.

![Execution time](/media/dashboard/dashboard-statement-plans-time.png)

> ### Coprocessor read

Reviewer: ```suggestion
### Coprocessor Read Tab
```

Click the **Coprocessor Read** tab, and you can see information related to Coprocessor read.

![Coprocessor read](/media/dashboard/dashboard-statement-plans-cop-read.png)

> ### Transaction

Reviewer: ```suggestion
### Transaction Tab
```

> Click the **Transaction** tab, and you can see information related to execution plan and transaction, such as the average number of written keys or the maximum number of written keys.

Reviewer 2: ```suggestion
Click the **Transaction** tab, and you can see information related to execution plans and transactions, such as the average number of written keys or the maximum number of written keys.
```

![Transaction](/media/dashboard/dashboard-statement-plans-transaction.png)

> ### Slow query

Reviewer: ```suggestion
### Slow Query Tab
```
